src/LanguageServer/Impl/Implementation/LineFormatter.cs

```diff
@@ -287,8 +287,15 @@ public class LineFormatter {
                         builder.EnsureEndsWithSpace();
                         break;
 
-                    case TokenKind.Name:
                     case TokenKind.Constant:
+                        if (token.IsString && next != null && next.IsString) {
```

Comment:
Does `next == null` means that it is the last token?
Also, do you need to check if they are on the same line?

Reply:
Yes, that `next` would be null if the current token were the last token on the line. That `next` is pulled from the list of tokens of the line, not the linked-list that's happening in `TokenExt` which could peer forward. I wouldn't want to look at the next line, since even if the next token (ignoring whitespace) happens to be a string literal, the fact that it's on another line implies that there's already spacing and adding a space here would only be adding trailing whitespace (which would then be trimmed away).